### PR TITLE
Nederdrone trim and INDI startup PD control

### PR DESCRIPTION
**Pull request by EwoudSmeur**
Added the option to set a desired value for the actuators other than zero, and to set priorities for the actuators.
Changed takeoff behavior: until in_flight is true no increments are added, turning the controller into a PD controller.

### CODE REVIEW

File: sw/airborne/firmwares/rotorcraft/stabilization/stabilization_indi.c [hunk 1]
```diff
@@ -440,73 +456,71 @@ void stabilization_indi_rate_run(struct FloatRates rate_sp, bool in_flight)
...
-          if (airspeed < 8.0) {
-            du_min[i] = GUIDANCE_INDI_MIN_THROTTLE - actuator_state_filt_vect[i];
-          } else {
-            du_min[i] = GUIDANCE_INDI_MIN_THROTTLE_FWD - actuator_state_filt_vect[i];
-          }
+    float airspeed = stateGetAirspeed_f();
+    //limit minimum thrust ap can give
+    if (!act_is_servo[i]) {
+      if ((guidance_h.mode == GUIDANCE_H_MODE_HOVER) || (guidance_h.mode == GUIDANCE_H_MODE_NAV)) {
+        if (airspeed < 8.0) {
```
Comment (OpenUAS) [inline sw/airborne/firmwares/rotorcraft/stabilization/stabilization_indi.c:489]: Magic number, a define would be better, reuse the AC define that are already in use like NOMINAL_AIRSPEED or MINIMUM_AIRSPEED

**Review by gautierhattenberger — DISMISSED** [review]
It can be merged like this, or with the suggestion to make the min airspeed  configurable as suggested

**Comment by EwoudSmeur** [comment]
I rebased on master to fix the merge conflicts.

**Comment by gautierhattenberger** [comment]
@EwoudSmeur I'm not sure if I should review this PR like this or not ? It has much more modifications for various elements than the initial one.

**Comment by EwoudSmeur** [comment]
Indeed, I made a mistake, one of the commits had all other changes that were on the Nederdrone branch and were not supposed to be there.

**Review by gautierhattenberger — APPROVED** [review]

**Pull request merged**